src/util/operations.tsx

```diff
@@ -5,9 +5,18 @@ export const getInstanceName = (operation?: LxdOperation): string => {
   // /1.0/instances/<instance_name>
   // /1.0/instances/<instance_name>?project=<project_name>
   // /1.0/instances/<instance_name>/snapshots/<snapshot_name>
+  // /1.0/instances/<instance_name>/<snapshot_name>
   return (
     operation?.resources?.instances
-      ?.filter((item) => item.startsWith("/1.0/instances/"))
+      ?.filter((item) => {
+        // for snapshots backend returns %2F instead of /, which is not decoded by the browser
+        const itemUrl = decodeURIComponent(item);
+        return (
+          itemUrl.startsWith("/1.0/instances/") &&
+          // exclude snapshots
+          itemUrl.split("/").length <= 4
+        );
```

Comment:
Can you link to the lxd issue reporting this bug so we can eventually clean this up when it is fixed there?

Reply:
Haven't created the issue yet, I will link it to a Jira issue later.